sirepo/package_data/denied_role_email.jinja

```diff
@@ -0,0 +1 @@
+Your request for {{ role }} access has been denied.
```

Comment:
missing end of line

Reply:
84a51a4c4d292b403b922c73411c9bad58de76bf